assets/sass/app.scss

```diff
@@ -32,6 +32,22 @@ footer.main-footer {
     font-size: 80%;
 }
 
+/* ================================ help ================================ */
```

Comment:
Can you create a new sass file for the documentation, as I like to improve the styles a little bit.
I think readability could be improved by increasing line height and some other font settings... but not in this PR, just so we have the sass file already in place.